client/client.go

```diff
@@ -546,9 +560,11 @@ func (client *Client) DebugGet(action string, result any, params map[string]stri
 }
 
 type defaultRequester struct {
-	baseURL   url.URL
+	baseURL   *url.URL
 	doer      doer
 	userAgent string
+	basicUser string
+	basicPass string
```

Comment:
nit:

```suggestion
	basicPassword string
```

Reply:
Fixed as part of the first username part. Resolving.